thoth/storages/graph/models.py

```diff
@@ -321,6 +346,11 @@ class HasArtifact(ReverseEdgeBase):
     """The given package has the given artifact."""
 
 
+@attr.s(slots=True)
+class InvestigatedBy(ReverseEdgeBase):
```

Comment:
This needs to be updated also in https://github.com/thoth-station/storages/pull/843

Reply:
It should be already updated - `PackageAnalyzer` run is connected using `Investigated` to `PythonArtifact` node.

Reply:
Shouldn't be ```investigated_by``` in schema?

Reply:
or this class should be ```Investigated```

Reply:
Actually should be ```Investigated```, otherwise the arrow should point to other direction.

Reply:
+1 to be consistent. What do you suggest? `Investigated` or `InvestigatedBy`?

Reply:
We used mainly active edge name for most of the nodes and especially for RUN components (e.g ```advised```, ```solved```, ```identified```, ```resolved```). So I would use ```Investigated``` in this case. Maybe it is worth creating a card for standardizing the semantic in Thoth Knowledge Graph because there are also other little inconsistencies.

Reply:
> We used mainly active edge name for most of the nodes and especially for RUN components (e.g `advised`, `solved`, `identified`, `resolved`). So I would use `Investigated` in this case. Maybe it is worth creating a card for standardizing the semantic in Thoth Knowledge Graph because there are also other little inconsistencies. (`analyzed_by` and `provided_by` for example)

+1, it would also worth to finally start using variables in the queries to avoid sql-injection-type attacks. So let's proceed with `investigated` now - which also keeps track of "data flow" in the system